plutus-benchmark/validation/Main.hs

```diff
@@ -19,17 +21,28 @@ import           Control.DeepSeq                          (force)
 import           Control.Monad.Trans.Except               (runExceptT)
 import qualified Data.ByteString                          as BS
 import qualified Data.ByteString.Lazy                     as BSL
-import           Data.List                                (sort)
+import           Data.List                                (isPrefixOf, sort)
 import           Flat
 import           System.Directory                         (listDirectory)
 import           System.FilePath
 
-{-- | Benchmarks based on validations obtained using
+{- | Benchmarks based on validations obtained using
 plutus-use-cases:plutus-use-cases-scripts, which runs various contracts on the
 blockchain simulator and dumps the applied validators as flat-encoded
 scripts. Generating these scripts is a very lengthy process involving building a
 lot of code, so the scripts were generated once and copied to the 'data'
-directory here.
+directory here.  Type 'cabal run plutus-use-cases:plutus-use-cases-scripts
+plutus-benchmark/validation/data' in the root directory of the Plutus repository
```

Comment:
There's an extra argument now.
```suggestion
plutus-benchmark/validation/data scripts' in the root directory of the Plutus repository
```

Reply:
I distinctly remember putting that in, but it seem to have mysteriously vanished again. I'll put it back and see if it stays there this time.